VCFoodDeliveryApp/Models/ApplicationUser.cs

```diff
@@ -12,5 +12,9 @@ public class ApplicationUser : IdentityUser
         public string LastName { get; set; }
         public int UsernameChangeLimit { get; set; } = 10;
         public byte[] ProfilePicture { get; set; }
+
+        public string Cart_Id { get; set; }
```

Comment:
Adding `Cart_Id` implies there is one Cart per ApplicationUser, but then you have a collection of Carts. If you mean to have many Carts per ApplicationUser, you should drop this.